docs/introduction-to-smart-contracts.rst

```diff
@@ -33,9 +33,9 @@ Storage
 The first line simply tells that the source code is written for
 Solidity version 0.4.0 or anything newer that does not break functionality
 (up to, but not including, version 0.5.0). This is to ensure that the
-contract does not suddenly behave differently with a new compiler version.
+contract does not suddenly behave differently with a new compiler version. For more information about pragma, see [here](https://en.wikipedia.org/wiki/Pragma_once).
```

Comment:
The link points to an article about one special kind of pragma, and solidity does not have this kind of "once" pragma.